### DISCUSSION TITLE
Filtering deprecated code

### DISCUSSION THREAD

**Discussion by crackedpotato007** [community/community#10753] [Copilot Conversations]
Hey! I have noticed co-pilot doesn't care about what  methods are supported in the currently installed version of the package, especially due to the introduction of discord.js V13 Most of the code available online is made with v12 which results in tens of errors because the code generated is full of deprecated methods

**Comment by dave9123** [comment]
You can manually add the intents and stuff tho

**Reply by crackedpotato007** [reply]
I am not talking about intents and stuff, plus DJS was only a example there are many depreceated stuff from V12 TO V13

**Reply by dave9123** [reply]
Oh then see google for some new changes

**Comment by crackedpotato007** [comment]
I can do them manually 🤣 that's why I made this thread so github co pilot respects package versions

**Comment by adnanjpg** [comment]
It would be cool if there was a way to report the deprecated APIs and suggest new ones
